internal/oc/release.go

```diff
@@ -0,0 +1,136 @@
+package oc
```

Comment:
how about adding tests to this model?

Reply:
I will add more tests in a followup PR.

Reply:
Why not in this one? 
We don't push code without test.

Reply:
Added unit test. PTAL.